ServiceWorker/index.ts

```diff
@@ -89,16 +89,17 @@ const httpTrigger: AzureFunction = async function (context: Context, req: HttpRe
 
     context.log(`Service Worker function is DONE processing a request for site: ${req.query.site}`);
   } catch (error) {
-    if (error.name && error.name.indexOf('TimeoutError') > -1) {
-
-      context.res = {
-        status: 500,
-        body: {
-          error: error
-        }
```

Comment:
This was returning a 500 error only if it was a timeout. Otherwise, we'd get a 200 response that contained an empty body.